UD-Viz-Core/examples/DemoBron/Demo.js

```diff
@@ -25,6 +25,8 @@ baseDemo.loadConfigFile('../data/config/generalDemoConfig.json').then(() => {
     const help  = new udvcore.HelpWindow();
     baseDemo.addModuleView('help', help);
 
+    baseDemo.config.server = baseDemo.config.servers["limonest_bron"];   
```

Comment:
Could you instead pass `baseDemo.config.servers["limonest_bron"]` (or the equivalent for other demos) directly to the modules that need the server configuration (authentication, documents, contribute, validation, document comments, guided tours and links etc.) since these modules don't seem to need other parts of the config file ?

Reply:
Do you mean something like that 
 *     new udvcore.AuthenticationService(requestService, baseDemo.config.servers["lyon"]);
Actually, I tried it and it seems that the link module needs to have access to all of the configuration file and I found no work around. Also, it would imply that we'll need to specify for each module the server needed.

Reply:
Ok, we'll leave it this way then